cli/src/test/java/com/devonfw/tools/ide/context/AbstractIdeContextTest.java

```diff
@@ -51,42 +51,47 @@ protected static IdeTestContext newContext(String projectName, String projectPat
   }
 
   /**
-   * @param projectName the (folder)name of the test project in {@link #PATH_PROJECTS}. E.g. "basic".
+   * @param projectTestCaseName the (folder)name of the project test case, in this folder a 'project' folder represents
+   *        the test project in {@link #PATH_PROJECTS}. E.g. "basic".
    * @param projectPath the relative path inside the test project where to create the context.
    * @param copyForMutation - {@code true} to create a copy of the project that can be modified by the test,
    *        {@code false} otherwise (only to save resources if you are 100% sure that your test never modifies anything
    *        in that project.
    * @return the {@link IdeTestContext} pointing to that project.
    */
-  protected static IdeTestContext newContext(String projectName, String projectPath, boolean copyForMutation) {
+  protected static IdeTestContext newContext(String projectTestCaseName, String projectPath, boolean copyForMutation,
+      boolean debug) {
```

Comment:
what is `debug` flag? No JavaDoc given.
Please always prefer to add a new method signature but keep the existing one to prevent merge conflicts with other feature branches, etc.
I will add this to our coding conventions as I am repeating this from one review to the next.

Reply:
That was just for testing, since somehow locally in windows the tests are green and the pipeline fails. The moving of files seems to have caused problems.